### PR TITLE
Added a failing test

### PR DESCRIPTION
**Pull request by Aribros**
Failing tests for #181

### CODE REVIEW

**Comment by charjr** [comment]
@Aribros What happens if there are duplicate operationIds in two separate paths?

[Unique string used to identify the operation. The id MUST be unique among all operations described in the API.](https://swagger.io/specification/#operation-object)

**Comment by Aribros** [comment]
I will make another commit to properly take care of this.

**Comment by Aribros** [comment]
> @Aribros What happens if there are duplicate operationIds in two separate paths?
> 
> [Unique string used to identify the operation. The id MUST be unique among all operations described in the API.](https://swagger.io/specification/#operation-object)

This has been taken care of. More tests were added to cover duplicate IDs in separate paths .

File: src/spec/Paths.php [hunk 1]
```diff
@@ -147,6 +151,18 @@ public function validate(): bool
...
+            foreach($operations as $operation) {
+                $id = $operation->operationId;
+                if(empty($id)) {
+                    continue;
+                }
+                if(!empty($this->_operationsIds[$id]) && $this->_operationsIds[$id]) {
+                    $this->_errors[] = "Operation ID '$id', already exist. Operation IDs must be unique.";
+                    $valid = false;
+                }
+                $this->_operationsIds[$id] = true;
```
Comment (cebe) [inline src/spec/Paths.php:164]: _operationsIds should be a local variable otherwise consecutive calls to validate() will report all ids as duplicate.

**Review by cebe — CHANGES_REQUESTED** [review]
